src/meta/meta_table.c

```diff
@@ -160,12 +194,11 @@ __wt_metadata_update(
 	if (WT_META_TRACKING(session))
 		WT_RET(__wt_meta_track_update(session, key));
 
-	WT_RET(__wt_metadata_cursor(session, "overwrite", &cursor));
+	WT_RET(__wt_metadata_cursor(session, &cursor));
```

Comment:
I'd rather keep this as an explicit overwrite setting, I guess there is an overhead to doing that, but relying on the default not changing makes me nervous.